openedx/features/caliper_tracking/transformers/navigation_transformers.py

```diff
@@ -15,3 +15,44 @@ def edx_ui_lms_link_clicked(current_event, caliper_event):
     :return: updated caliper_event.
     """
     return current_event
+
+
+def edx_course_tool_accessed(current_event, caliper_event):
+    """
+        The browser emits an edx.course.tool.accessed event when a user clicks
```

Comment:
Un-indent this to make it exactly below the above `"""`

Reply:
fixed